ldms/src/ldmsd/ldmsd_stream.c

```diff
@@ -603,7 +603,7 @@ int ldmsd_stream_publish_file(const char *stream, const char *type,
 {
 	int rc, stream_type;
 	size_t data_len;
-	static char *buffer;
+	static char *buffer; /* why is buffer static? not reused */
```

Comment:
We should remove that comment. If we know it's not reused, why add that in a comment and leave the bug there?

Reply:
removed 'static' and comment.